README.md

```diff
@@ -1,82 +1,67 @@
-        }
-      }
-    }
-  }
-}
-```
-
-## Configure Webpack
-
-Alias `react-native` with `@open-native/core`.
-
-```js
-webpack.chainWebpack((config) => {
-  config.resolve.alias.set('react-native', '@open-native/core');
-});
-```
-
-If you install a react-natie module & get errors in webpack, install `metro-react-native-babel-preset` via npm/yarn.
-
-```
-npm install -D metro-react-native-babel-preset
-```
-
-add the following preset to your webpack config.
-
-```js
-webpack.chainWebpack((config) => {
-  config.resolve.alias.set('react-native', '@open-native/core');
-  config.module
-    .rule('rnmodules')
-    // Add each react-native module that gives errors in webpack build here
-    // like this:
-    .include.add(/node_modules(.*[/\\])+react-native-module/)
-    .end()
-    .use('babel-loader')
-    .before('ts-loader')
-    .loader('babel-loader')
-    .options({
-      babelrc: false,
-      presets: ['module:metro-react-native-babel-preset'],
-    });
-});
-```
-
-That's all, try out some react-native modules & feel free to open and issue if you face a problem.
-
-Join the [NativeScript Discord Community](https://discord.com/invite/RgmpGky9GR)
-
-## Contributing
-
-If you want to contribute, feel free to open an issue or join our discord community to discuss your ideas.
-
-# MIT License
-
-Copyright (c) 2022 Ammar Ahmed
+# Introducing Open Native: Native Modules, for all
+
+We all want to build apps in a way that's approachable to us, regardless of the target platform. Projects like React Native, Flutter, Capacitor, and NativeScript enable us to build native (e.g. iOS and Android) apps using alternative idioms such as JavaScript, Web tech, or platform-agnostic UI.
+
+Each of these projects have a way to map platform APIs into their idiom (e.g. React Native has "native modules"), but none are completely mutually compatible. That is to say, a React Native native module cannot be used in a Flutter app as-is, and vice versa. This situation has led to a great amount of duplicated effort, and an isolation of communities.
+
+[Open Native](https://github.com/OpenNative/open-native) to the rescue.
+
+<p align="center">
+  <img src="open-native-logo.png" width="270">
+</p>
+
+<p align="center">
+  <a href="https://www.npmjs.com/package/@open-native/core">
+    <img src="https://badge.fury.io/js/@open-native%2Fcore.svg" alt="npm version" height="18">
+  </a>
+  <a href="https://opensource.org/licenses/mit-license.php" alt="MIT licence">
+    <img src="https://badges.frapsoft.com/os/mit/mit.png?v=103"/>
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=ammarahm_ed" alt="Follow Ammar Ahmed on Twitter">
+    <img src="https://img.shields.io/twitter/follow/ammarahm_ed.svg?style=social&logo=twitter"/>
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=LinguaBrowse" alt="Follow Jamie Birch on Twitter">
+    <img src="https://img.shields.io/twitter/follow/LinguaBrowse.svg?style=social&logo=twitter"/>
+  </a>
+</p>
```

Comment:
These badges take up more horizontal space than I'd anticipated. I'd be happy to move them to the bottom instead.